src/main/java/com/tterrag/registrate/util/DataIngredient.java

```diff
@@ -15,6 +15,7 @@
 import net.minecraft.advancements.critereon.ItemPredicate;
 import net.minecraft.network.FriendlyByteBuf;
 import net.minecraft.resources.ResourceLocation;
+import net.minecraft.tags.Tag;
```

Comment:
Is this necessary?

Reply:
Huh, it looks like my IDE imported this for some damn reason, will fix it in a minute